pkg/network/usm/ebpf_gotls.go

```diff
@@ -288,6 +288,7 @@ func (p *GoTLSProgram) Start() {
 }
 
 func (p *GoTLSProgram) Stop() {
+	close(p.done)
 	if p.procMonitor.cleanupExec != nil {
```

Comment:
we should wait the main loop here
	p.wg.Wait()
or we may portentially have a race at exit (processSync.C running and close(p.done) called, so we have
monitor.FindDeletedProcesses()  p.unregisterProcess() on one goroutine potentially running at the same time than
p.procMonitor.cleanupExec() p.procMonitor.cleanupExit()  p.unregisterProcess(pid) (not under lock)